src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs

```diff
@@ -268,7 +268,7 @@ public override void ExecuteCmdlet()
             return;
         }
 
-        public object GetDynamicParameters()
+        public new object GetDynamicParameters()
```

Comment:
`new` must be added since there are build error (actually warning, but we treat warning as error), but there are no compatibility issues at run time.